Synthesis failed for python-speech

Hello! Autosynth couldn't regenerate python-speech. :broken_heart:

Here's the output from running `synth.py`:

```
rpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (49.6.0)
Requirement already satisfied: google-auth<2.0dev,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-nc8fg8mu/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-apzjk7ta/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-nc8fg8mu/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-apzjk7ta/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-nc8fg8mu/pyaudio/
You are using pip version 18.1, however version 20.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-04 06:30:43,215 autosynth [ERROR] > Synthesis failed
2020-09-04 06:30:43,215 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-04 06:30:43,229 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-04 06:30:43,237 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/81fde0ed-dbce-460b-b114-ad0fb8796a90/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
(from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (49.6.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: pytz in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-4p6x7krt/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-lph6wl64/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-4p6x7krt/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-lph6wl64/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-4p6x7krt/pyaudio/
You are using pip version 18.1, however version 20.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-05 06:32:01,035 autosynth [ERROR] > Synthesis failed
2020-09-05 06:32:01,035 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-05 06:32:01,049 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-05 06:32:01,058 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/61e7f123-4bf9-4176-a25d-75d07da372eb/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: pytz in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-2u7fso1u/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-0_zqmecv/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-2u7fso1u/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-0_zqmecv/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-2u7fso1u/pyaudio/
You are using pip version 18.1, however version 20.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-06 06:28:56,228 autosynth [ERROR] > Synthesis failed
2020-09-06 06:28:56,228 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-06 06:28:56,241 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-06 06:28:56,249 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/05f5f38a-a10a-4623-9963-ba766645d417/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (49.6.0)
Requirement already satisfied: google-auth<2.0dev,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.1)
Requirement already satisfied: protobuf>=3.12.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.13.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-606_r664/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-v6m4gs53/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-606_r664/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-v6m4gs53/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-606_r664/pyaudio/
You are using pip version 18.1, however version 20.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-07 06:26:18,561 autosynth [ERROR] > Synthesis failed
2020-09-07 06:26:18,561 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-07 06:26:18,574 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-07 06:26:18,582 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/60669ff0-673a-4932-ac4b-a4dd666115cc/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
rpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: google-auth<2.0dev,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-276jet1v/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-dvkoahmt/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-276jet1v/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-dvkoahmt/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-276jet1v/pyaudio/
You are using pip version 18.1, however version 20.2.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-08 06:29:35,504 autosynth [ERROR] > Synthesis failed
2020-09-08 06:29:35,505 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-08 06:29:35,517 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-08 06:29:35,524 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ed4eb6af-a475-4d3c-b359-9abf2e8dffea/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.31.0)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-j27_qzah/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-0ouj83tn/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-j27_qzah/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-0ouj83tn/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-j27_qzah/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-09 06:32:49,019 autosynth [ERROR] > Synthesis failed
2020-09-09 06:32:49,019 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-09 06:32:49,032 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 06:32:49,040 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6119a864-2de4-4996-b2e7-89215093ff0a/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
>google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.13.0)
Requirement already satisfied: google-auth<2.0dev,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.1)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-qcbcgvex/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-u6pmp29c/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-qcbcgvex/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-u6pmp29c/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-qcbcgvex/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-10 06:29:22,914 autosynth [ERROR] > Synthesis failed
2020-09-10 06:29:22,914 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-10 06:29:22,928 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 06:29:22,936 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f682beb7-6c7c-4fc2-8bbd-d95360cbd659/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
pi-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: protobuf>=3.12.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.13.0)
Requirement already satisfied: setuptools>=34.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-c5w2gcqr/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-aq0ixcgk/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-c5w2gcqr/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-aq0ixcgk/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-c5w2gcqr/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-11 06:42:06,903 autosynth [ERROR] > Synthesis failed
2020-09-11 06:42:06,903 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-11 06:42:06,918 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 06:42:06,930 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/eb7165ef-a82f-4002-b64d-f55176216eb7/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
v,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-mmt4bl6i/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-hr1pdt1h/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-mmt4bl6i/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-hr1pdt1h/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-mmt4bl6i/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-12 06:28:07,624 autosynth [ERROR] > Synthesis failed
2020-09-12 06:28:07,624 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-12 06:28:07,637 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 06:28:07,645 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/debb771f-50f3-4a16-83b5-2db1bf4b6dc0/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.1)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-_vnrast7/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-clgz32it/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-_vnrast7/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-clgz32it/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-_vnrast7/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-13 06:24:45,461 autosynth [ERROR] > Synthesis failed
2020-09-13 06:24:45,462 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-13 06:24:45,477 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 06:24:45,486 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/16b74b0d-9789-4ccd-b420-8e8b743095a9/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
pi-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.13.0)
Requirement already satisfied: setuptools>=34.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-73nfojb3/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-c9op2j27/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-73nfojb3/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-c9op2j27/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-73nfojb3/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-14 06:23:42,732 autosynth [ERROR] > Synthesis failed
2020-09-14 06:23:42,732 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-14 06:23:42,745 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 06:23:42,753 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/50e01ce5-43b0-42dd-ad30-2a49163d3459/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
rom google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: setuptools>=34.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: pytz in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-tt1pjh9_/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-5pswv7qu/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-tt1pjh9_/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-5pswv7qu/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-tt1pjh9_/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-15 06:21:55,024 autosynth [ERROR] > Synthesis failed
2020-09-15 06:21:55,024 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-15 06:21:55,037 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 06:21:55,045 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ae69e792-ad74-42fa-ada8-d5be248bc596/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
pi-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: setuptools>=34.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-s8kcj79r/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-cal7gtl8/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-s8kcj79r/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-cal7gtl8/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-s8kcj79r/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-16 06:19:57,012 autosynth [ERROR] > Synthesis failed
2020-09-16 06:19:57,012 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-16 06:19:57,025 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 06:19:57,034 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d07f7197-065b-43d4-917b-d318d4843e72/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-eajkup5o/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-awdyed2u/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-eajkup5o/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-awdyed2u/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-eajkup5o/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-17 06:12:44,935 autosynth [ERROR] > Synthesis failed
2020-09-17 06:12:44,935 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-17 06:12:44,948 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 06:12:44,957 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/populate-secrets.sh
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/45c1b673-c8fe-4e90-b42e-5b4cf0dce448/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-1532qkbz/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-j45b3pl3/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-1532qkbz/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-j45b3pl3/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-1532qkbz/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-18 06:09:28,662 autosynth [ERROR] > Synthesis failed
2020-09-18 06:09:28,662 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-18 06:09:28,674 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-18 06:09:28,682 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/populate-secrets.sh
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b62b5851-de86-4dda-8948-a8d4d8874d59/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: google-auth<2.0dev,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.21.2)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-89t1jsg7/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-orc5nzo8/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-89t1jsg7/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-orc5nzo8/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-89t1jsg7/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-19 06:06:38,967 autosynth [ERROR] > Synthesis failed
2020-09-19 06:06:38,967 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-19 06:06:38,976 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-19 06:06:38,982 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/populate-secrets.sh
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/db819702-6edf-4a5b-9b35-fb99f484575c/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
>-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.1)
Requirement already satisfied: setuptools>=34.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: pyasn1>=0.1.3 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from rsa<5,>=3.1.4; python_version >= "3.5"->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-u0qb0wyc/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-4ykoif3x/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-u0qb0wyc/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-4ykoif3x/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-u0qb0wyc/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-20 06:04:48,513 autosynth [ERROR] > Synthesis failed
2020-09-20 06:04:48,513 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-20 06:04:48,523 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-20 06:04:48,530 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/populate-secrets.sh
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9822fc6a-3b76-40f6-8dd2-3fab1f13a390/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

Autosynth is still having trouble generating python-speech. :sob:

Here's the output from running `synth.py`:

```
2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (50.3.0)
Requirement already satisfied: requests<3.0.0dev,>=2.18.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.24.0)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.52.0)
Requirement already satisfied: grpcio<2.0dev,>=1.29.0; extra == "grpc" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.32.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.1.1)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3.5" in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (4.6)
Requirement already satisfied: chardet<4,>=3.0.2 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from requests<3.0.0dev,>=2.18.0->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (1.25.10)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /tmpfs/src/github/synthtool/env/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.21.1->google-api-core[grpc]<2.0.0dev,>=1.14.0->google-cloud-speech==1.3.2->-r /home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt (line 1)) (0.4.8)
Installing collected packages: pyaudio
  Running setup.py install for pyaudio: started
    Running setup.py install for pyaudio: finished with status 'error'
    Complete output from command /tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-6xcrkcay/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-tovpkb9_/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying src/pyaudio.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_portaudio' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmpfs/src/github/synthtool/env/include -I/home/kbuilder/.pyenv/versions/3.6.9/include/python3.6m -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.6/src/_portaudiomodule.o
    src/_portaudiomodule.c:29:23: fatal error: portaudio.h: No such file or directory
     #include "portaudio.h"
                           ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/tmpfs/src/github/synthtool/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmpfs/tmp/pip-install-6xcrkcay/pyaudio/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmpfs/tmp/pip-record-tovpkb9_/install-record.txt --single-version-externally-managed --compile --install-headers /tmpfs/src/github/synthtool/env/include/site/python3.6/pyaudio" failed with error code 1 in /tmpfs/tmp/pip-install-6xcrkcay/pyaudio/
You are using pip version 18.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-speech/synth.py", line 84, in <module>
    python.py_samples()
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 132, in py_samples
    sample_readme_metadata = _get_sample_readme_metadata(sample_project_dir)
  File "/tmpfs/src/github/synthtool/synthtool/languages/python.py", line 85, in _get_sample_readme_metadata
    shell.run([sys.executable, "-m", "pip", "install", "-r", requirements])
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'pip', 'install', '-r', '/home/kbuilder/.cache/synthtool/python-speech/samples/microphone/requirements.txt']' returned non-zero exit status 1.
2020-09-21 06:05:11,986 autosynth [ERROR] > Synthesis failed
2020-09-21 06:05:11,986 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at abc7d52 chore: fix lint
2020-09-21 06:05:11,995 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-21 06:05:12,002 autosynth [DEBUG] > Running: git clean -fdx
Removing .kokoro/populate-secrets.sh
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/baf58548-84cb-4703-92e9-bfbd904043f5/targets/github%2Fsynthtool;config=default/tests;query=python-speech;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -81,7 +81,7 @@ s.move(templated_files)
 # Samples templates
 # ----------------------------------------------------------------------------
 
-python.py_samples()
+python.py_samples(skip_readmes=True)
 
 
 # TODO(busunkim): Use latest sphinx after microgenerator transition

